tests/Sdk/Unit/Trace/ExporterFactoryTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Tests\Sdk\Unit\Trace;
+
+use OpenTelemetry\Contrib as Path;
+use OpenTelemetry\Sdk\Trace\ExporterFactory as ExporterFactory;
+use PHPUnit\Framework\TestCase;
+
+class ExporterFactoryTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function testIfExporterHasCorrectEndpoint()
+    {
+        $input = 'zipkin+http://zipkin:9411/api/v2/spans';
+        $factory = new ExporterFactory('test.zipkin');
+        $exporter = $factory->fromConnectionString($input);
+        $this->assertInstanceOf(Path\Zipkin\Exporter::class, $exporter);
+        
+        $input = 'jaeger+http://jaeger:9412/api/v2/spans';
+        $factory = new ExporterFactory('test.jaeger');
+        $exporter = $factory->fromConnectionString($input);
+        $this->assertInstanceOf(Path\Jaeger\Exporter::class, $exporter);
+
+        $input = 'newrelic+https://trace-api.newrelic.com/trace/v1?licenseKey="23423423';
+        $factory = new ExporterFactory('test.newrelic');
+        $exporter = $factory->fromConnectionString($input);
+        $this->assertInstanceOf(Path\Newrelic\Exporter::class, $exporter);
+
+        $input = 'otlp+';
+        $factory = new ExporterFactory('test.otlp');
+        $exporter = $factory->fromConnectionString($input);
+        $this->assertInstanceOf(Path\Otlp\Exporter::class, $exporter);
+
+        $input = 'otlpgrpc+';
```

Comment:
maybe `otlp+grpc://` here?

Reply:
The way that the input string is being parsed is by separating the contrib type from the baseUrl by '+' 
Since there is an exporter `OtlpGrpc`, I created a case where `otlpgrpc` is a valid contrib type.
What alternatives could I look into ?

Reply:
The connection string doesn't need to directly represent the implementation, it's more of an API. If in the future the OTLP exporters get merged into a single class for some reason (or renamed), you wouldn't want to change the connection string.

[The idea is to represent both `protocol+transport://`](https://github.com/whatwg/url/issues/230), with the transport maybe being optional if it's only one possible. For example, if Zipkin works only over HTTPS, we could in theory have `zipkin://` and `zipkin+https://` be synonymous.

In this case, the protocol is `otlp`, the transport is either `https` or `grpc`. Once of them (the dominant one) might be the implied default, but we could also disallow that and have them be mandatory.

For simplicity sake:
- schema is `<protocol>+<transport>://`
- some protocols have more than one format, like `otlp`
- the connection string is not expected to 100% align with the implementation, especially since it'll once (hopefully) be standardized by the spec (where the exporter naming schemes might be completely different